scripts/cross/bootstrap-ubuntu.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/sh
 
 apt-get update
-
```

Comment:
Simply made to trigger the "Cross" workflow steps.. but it's just whitespace changing so who cares.